clarity/src/vm/analysis/type_checker/v2_1/natives/mod.rs

```diff
@@ -699,6 +700,48 @@ fn check_get_burn_block_info(
     )?)
 }
 
+fn check_get_stacks_block_info(
+    checker: &mut TypeChecker,
+    args: &[SymbolicExpression],
+    context: &TypingContext,
+) -> TypeResult {
+    check_arguments_at_least(2, args)?;
+
+    let block_info_prop_str = args[0].match_atom().ok_or(CheckError::new(
+        CheckErrors::GetStacksBlockInfoExpectPropertyName,
+    ))?;
+
+    let block_info_prop =
+        StacksBlockInfoProperty::lookup_by_name(block_info_prop_str).ok_or(CheckError::new(
+            CheckErrors::NoSuchStacksBlockInfoProperty(block_info_prop_str.to_string()),
+        ))?;
+
+    checker.type_check_expects(&args[1], context, &TypeSignature::UIntType)?;
+
+    Ok(TypeSignature::new_option(block_info_prop.type_result())?)
+}
+
+fn check_get_tenure_info(
+    checker: &mut TypeChecker,
+    args: &[SymbolicExpression],
+    context: &TypingContext,
+) -> TypeResult {
+    check_arguments_at_least(2, args)?;
```

Comment:
`check_argument_count`

Reply:
Fixed in 1fad8d226e8081b4204c6a6ecc44c5ff8038013b